octavia-cli/octavia_cli/templates/connection.yaml.j2

```
@@ -0,0 +1,42 @@
+name: {{ resource_name }}
+source_id: {{ source_id }}
+destination_id: {{ destination_id }}
```

```suggestion
resource_name: mock_connection
definition_type: connection
source_id: mock_source
destination_id: mock_destination

```
Could you please try to mimic the top of the [other template?](https://github.com/airbytehq/airbyte/blob/master/octavia-cli/octavia_cli/templates/source_or_destination.yaml.j2)

---

Maybe we can create template composition with Jinja to make `source_or_destination.yaml.j2` and `connection.yaml.j2` share the same headers.